### ISSUE TITLE
Add numeric fields to solr

### ISSUE THREAD

**Issue by cdrini** [internetarchive/openlibrary#5133]
Part of #2728 . Blocked by solr 8 deploy.

eg Counts (Reading Log, Lists, Stars), List membership, Do-We-Want-It, Partner collections. Exact list to be determined.

**Comment by cdrini** [comment]
Closing in favour of #3990 ; that one is more scoped.
